CONTRIBUTING.md

```diff
@@ -55,3 +55,27 @@ Compared to our earlier CLA, DCO tends to make the experience simpler for new
 contributors. If you are contributing as an employee, there is no need for your
 employer to sign anything; the DCO assumes you are authorized to submit
 contributions (it's your responsibility to check with your employer).
+
+## Building and Testing your changes
+
+### Dependencies
+
+* [Go >= 1.19](https://golang.org/doc/install)
+* [Yarn >= 1.22](https://classic.yarnpkg.com/lang/en/docs/install/)
+
+### Unit Tests
+
+To run unit tests for the backend, use `make test`.
+
+To run unit tests for the frontend, use `cd client/web/antrea-ui/ && yarn test`.
```

Comment:
I got an error like below for the first try:
```
$ react-scripts test
/bin/sh: react-scripts: command not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command
```
Maybe remind user to run `npm install` first?

Reply:
you're right (`yarn install` in this case), I updated the doc